user/user.go

```diff
@@ -12,6 +12,7 @@ type User struct {
 	Id       string                 `json:"userid" bson:"userid,omitempty"` // map userid to id
 	Name     string                 `json:"username" bson:"username"`
 	Emails   []string               `json:"emails" bson:"emails"`
+	Terms    string                 `json:"terms" bson:"terms"`
```

Comment:
It isn't clear to me what `Terms` means.  After I read the rest of the code I thought `TermsAccepted` might be clearer.